back/engines/commercial/flag_inappropriate_content/spec/serializers/flag_inappropriate_content/web_api/v1/notifications/inappropriate_content_flagged_serializer_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe FlagInappropriateContent::WebApi::V1::Notifications::InappropriateContentFlaggedSerializer do
+  it 'includes a relative path to the respective Idea' do
+    flagged = create(:inappropriate_content_flagged)
+    hash = described_class.new(flagged).serializable_hash
+    slug = Idea.find(flagged.inappropriate_content_flag.flaggable.post_id).slug
+
+    expect(hash[:data][:attributes][:flaggable_path]).to eq("ideas/#{slug}")
```

Comment:
This is not wrong, happy you covered it with a spec, but just a small piece of advice:

It's often better to not use variables in your expectation checks when you can avoid them. In this case, by checking it equals `ideas/#{slug}` your test become a bit less robust. Suppose for example the inappropriate_content_flagged model changes in some way, and we rename `slug` to `plug`. This test would still succeed, as it would render as `ideas/nil`.

The alternative is that you actively set the slug to a specific string when you create the data, and then check that exact string in your assertion, that will never succeed by coincidence.

Reply:
Good catch. I completely forgot about the `nil` thing. Thanks! 👍